internal/tui/components/inputbox/inputbox.go

```diff
@@ -110,3 +165,31 @@ func (m *Model) UpdateProgramContext(ctx *context.ProgramContext) {
 	m.ctx = ctx
 	m.inputHelp.Styles = ctx.Styles.Help.BubbleStyles
 }
+
+func extractCurrentLabel(input string) string {
```

Comment:
this seems to be a duplicate. I think you should delete the one in `autocomplete` as it doesn't need to know about the commas.